pycalphad/tests/test_equilibrium.py

```diff
@@ -224,20 +238,22 @@ def test_eq_ternary_edge_misc_gap():
                  [0.33366666666666667, 0.44455555555555554, 0.22177777777777785]
     assert np.all(np.abs(mass_error) < 0.001)
 
+@pytest.mark.solver
 def test_eq_issue43_chempots_misc_gap():
     """
     Equilibrium for complex ternary miscibility gap (gh-43).
     """
     eq = equilibrium(ISSUE43_DBF, ['AL', 'NI', 'CR', 'VA'], 'GAMMA_PRIME',
                      {v.X('AL'): .1246, v.X('CR'): 1e-9, v.T: 1273, v.P: 101325},
                      verbose=True)
-    chempots = 8.31451 * np.squeeze(eq['T'].values) * np.array([-19.47631644, -25.71249032,  -6.0706158])
-    mass_error = np.nansum(np.squeeze(eq.NP * eq.X), axis=-2) - \
-                 [0.1246, 1e-9, 1-(.1246+1e-9)]
-    assert np.max(np.fabs(mass_error)) < 1e-9
+    chempots = np.array([-206144.57, -272150.79, -64253.652])
+    # TODO: This change needs discussion
+    # Do we care if the absolute mass error in the Ni amount is 2e-9, instead of 1e-9?
+    assert_allclose(np.nansum(np.squeeze(eq.NP * eq.X), axis=-2), [0.1246, 1e-9, 1-(.1246+1e-9)], rtol=1e-5)
```

Comment:
Is the comment here out of date?

I think the `rtol=1e-5` here is acceptable.

Reply:
Removed comment